visualizer/src/TimeLineConfig.h

```diff
@@ -23,7 +23,7 @@ SOFTWARE. */
 #include <QColor>
 
 namespace visualizer {
-    using zoom_t = unsigned long int;
+    using zoom_t = long long int;
```

Comment:
my ubuntu build shows **long int** and **long long int** actually being same size of 8, is it different on your platform? Does type need to be signed? Just checked gcc10 for __uint128_t seems to be 16 byte and probably do the trick.

Reply:
According to https://en.cppreference.com/w/cpp/language/types unsigned long int can be 32 bit on some systems.
Good catch, its not needed to be signed, probably will update to unsigned long long int